src/helpers/markdown-helpers.js

```diff
@@ -130,11 +199,17 @@ const generateSummaryFile = (content, summaryFileName, parentDirectory = '', cur
       return;
     }
     if (item.type === 'directory') {
-      const link = path.join(parentDirectory, item.name, `${item.name}.md`);
+      const link = createFileLink ? path.join(parentDirectory, item.name, `${item.name}.md`) : `#${item.name}`;
```

Comment:
@Incee That's where I am not sure how the link should look like? I know we have to use an anchor, but I did not manage to create working anchor, at least in vscode. I'd like to see an example of how you did it so that I can adapt the code

Reply:
I will check with Sulav and ask him to support here, as he wrote the script...

Reply:
@Sulav1

Reply:
@msevestre I used syntax [toc]. When the markdown doc is opened in the typora, it generates the table of content.